proposals/wac-ucr/index.bs

```diff
@@ -291,6 +296,26 @@ He can't see the contents of the [=resources=], or remove anything in the list.
 That's fine because he only needs to add the [=resources=] that aren't
 included.
 
+### Read-append-delete access to a Collection ### {#collection-readappenddelete}
+
+Note: Despite this section's focus on permissions specific to the
+collection resource itself, this use case also incorporates inherited read
+access to better illustrate the scenario.
+
+Over time Alice has asked her colleagues to leave comments on her `portfolio`,
+which are stored in the `comments` [=collection=].
+
+When she asks a colleague to leave a comment, she gives them the following
+permissions:
+
+* Inherited [=read access=] to the `comments` [=collection=]
+    so they can list and read existing comments (including their own).
+* [=Append access=] to the `comments` [=collection=], which allows them to
+    add their own comments, but not change anyone else's including their
+    own after the comments have been submitted.
+* Ability to delete any resources in `comments` that they
+    have created themselves, but none created by anyone else.
+
```

Comment:
In this use case I don't see delete only, rather than write, justified. I would propose to use *write* here and add another use case which would justify delete only requirement.

Reply:
I think a use case for creation of a post and allowing the author to write for a while, then only to delete would be good. This use case could then refer to that new use case.

Reply:
@elf-pavlik i believe this is resolved now with the latest changes.